instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/JettyServlet3Test.groovy

```diff
@@ -149,6 +149,11 @@ class JettyServlet3TestAsync extends JettyServlet3Test {
   boolean isAsyncTest() {
     true
   }
+
+  @Override
+  String getMetricsInstrumentationName() {
+    "io.opentelemetry.servlet-3.0"
+  }
```

Comment:
I couldn't think why metric has different instrumentation name from the SERVER span in these cases? thx